src/components/LibrariesListItem.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from "react";
+import { Panel } from "react-bootstrap";
+import { LibraryData } from "../interfaces";
+import LibraryDetailContainer from "./LibraryDetailContainer";
+import { GenericWedgeIcon } from "@nypl/dgx-svg-icons";
+
+export interface LibrariesListItemProps {
+  library: LibraryData;
+  active: boolean;
+  select: (event: __React.MouseEvent) => void;
+}
+export interface LibrariesListItemState {
+  open: boolean;
+}
+
+export default class LibrariesListItem extends React.Component<LibrariesListItemProps, LibrariesListItemState> {
+  constructor(props: LibrariesListItemProps) {
+    super(props);
+    this.colorCode = this.colorCode.bind(this);
+  }
+
+  header() {
+    return (
+      <div onClick={this.props.select}>
```

Comment:
`<div>`s should not have an `onClick` attribute unless if you give it a `role="button"` attribute. In that case, you might as well use a `<button>`.